src/components/DropzoneAreaBase.js

```diff
@@ -87,6 +87,14 @@ class DropzoneAreaBase extends React.PureComponent {
         snackbarVariant: 'success',
     };
 
+    componentDidUpdate() {
```

Comment:
I'd prefer to avoid using `componentDidUpdate` to implement this feature and instead I'd use a callback method when using the `setState` on the `SnackbarAlert` status, this because it would make the flow more clear and avoid adding checks during the `componentDidUpdate` (which is triggered for many reasons other than the actual state updates).

You can have a look at how `notifyFileChange` has been implemented on `DropzoneArea` component for reference.

Reply:
I take your point on avoiding additional checks. I disagree with the flow, as if a new snackbar message were to be added, we will no longer automatically pick it up with the dev knowing to also cater for onAlert.

With the switch to hooks, this should definitely be in a useEffect(onAlert, [message, variant]) so that we always pick up and snackbar messages and pass them on.